client/components/Dashboard.js

```diff
@@ -0,0 +1,181 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { fetchMeetups } from '../store';
+import { Redirect } from 'react-router-dom';
+// import * as _ from 'lodash';
+import axios from 'axios';
+// import {daysInMonth} from '../../utils';
+
+class Dashboard extends Component {
+    constructor(props){
+        super();
+        this.state = {
+        	userApi: {},
+        	//put all of this user's meetups in the local state
+        	meetupIdArray: [],
+          meetupIncludeUserArray: []
+
+        };
+        // this.handleSubmit = this.handleSubmit.bind(this);
+        // this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleSubmit(event) {
+    }
+
+    handleChange(event) {
+    }
+    
+    componentDidMount(props){
+    	const userId = this.props.user.id
+    	//bad to do this directly in here instead of putting on store?
+    	axios.get(`/api/users/${userId}`)
```

Comment:
Ideally this would go into the store as to not overcomplicate the component

Reply:
agreed